indico/web/client/js/jquery/widgets/jinja/permissions_widget.js

```diff
@@ -479,9 +461,67 @@ import Palette from 'indico/utils/palette';
       });
 
       // Manage the addition to users/groups to the acl
-      $('.js-add-user-group').on('click', () => {
-        this._addUserGroup();
-      });
+      const userSearchTrigger = triggerProps => (
+        <a className="i-button" {...triggerProps}>
+          <Translate>User</Translate>
+        </a>
+      );
+
+      const groupSearchTrigger = triggerProps => (
+        <a className="i-button" {...triggerProps}>
+          <Translate>Group</Translate>
+        </a>
+      );
+
+      const existing = JSON.parse(this.element.find('input[name="permissions"]').val()).map(
+        e => e[0].identifier
+      );
+
+      ReactDOM.render(
+        <>
+          <PersonLinkSearch
```

Comment:
In the end, `PersonLinkSearch` is only wrapping `UserSearch` and including the `useFavoriteUsers` hook. I think we could just get rid of `PersonLinkSearch` altogether and get the favorite users in the two places where `<PersonLinkSearch` is currently used and use `UserSearch` directly.